pkg/cmd/flags.go

```diff
@@ -20,3 +21,37 @@ func ValidateUpdateSliceFlags(cmd *cobra.Command, args []string) error {
 	}
 	return nil
 }
+
+const (
+	flagDoRunMigrations = "do-migrations"
+	flagSkipMigrations  = "skip-migrations"
+)
+
+func SetMigrationFlags(cmd *cobra.Command) {
+	cmd.Flags().Bool("do-migrations", true, "Specify if you want to force migrations to run")
+	cmd.Flags().Bool("skip-migrations", false, "Specify if you want to skip migrations")
+}
+
+func ShouldRunMigrations(cmd *cobra.Command, happyConf *config.HappyConfig) (bool, error) {
+	if cmd.Flags().Changed(flagDoRunMigrations) && cmd.Flags().Changed(flagSkipMigrations) {
+		return false, errors.Errorf(
+			"Flags %s and %s cannot be specified at the same time",
```

Comment:
I think conventionally we shouldn't start errors with capital letters since they might be wrapped by other errors

Reply:
I thought the linter caught these